### DISCUSSION TITLE
How to Upload Past Spot Data to PVOutput

### DISCUSSION THREAD

**Discussion by jtj02** [SBFspot/SBFspot#709] [Q&A] [answered]
Hi, I have a few months of generated spot data in the sqlite3 database that I would like to manually add to me system in PVOutput. I can see comments etc in the code that there is some path for generating the data but I cannot see exactly how to do this. Can anyone tell me how or point me in the right direction?

The background is that I installed SBFspot on my RPI some months ago but the UploadDaemon was not working. I kicked it somehow and it started (might have been by setting the time on my old SMA inverter). So I have two days data in PVOutput but now would like to get the past few months up there as well.

**Comment by sbf-** [comment, marked as answer]
There is a PVoutput limit of 14 days (90 days in donation mode) This means you can't upload older spotdata. However, manual upload of monthdata is possible.
If the spotdata is recent, the upload daemon will do this automatically

**Reply by jtj02** [reply]
Thanks. Older monthly data has not been uploaded, I do not think monthly data has been generated from spot data in the database. Is there a manual command where I can do this? My PVOutput login is "HillviewRd" and looking at monthly data there only shows partial month for September with the two days that have been uploaded so far.
